modules/catalog/src/main/java/org/apache/ignite/internal/catalog/descriptors/CatalogZoneDescriptor.java

```diff
@@ -44,19 +44,20 @@ public class CatalogZoneDescriptor extends CatalogObjectDescriptor {
     private final String filter;
 
     /** Data storage descriptor. */
-    private CatalogDataStorageDescriptor dataStorage;
+    private final CatalogDataStorageDescriptor dataStorage;
 
     /**
      * Constructs a distribution zone descriptor.
      *
      * @param id Id of the distribution zone.
      * @param name Name of the zone.
-     * @param partitions Number of partitions in distributions zone.
-     * @param replicas Number of partition replicas.
+     * @param partitions Amount of partitions in distributions zone.
+     * @param replicas Amount of partition replicas.
```

Comment:
"Amount" is only used for uncountable stuff, right? Like "the amount of water", where you can't say "2 waters". But you can say 2 partitions or 2 replicas. I believe you're wrong here. We may consult folks who know English better that I do to be confident.

Reply:
fix it